<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Downloads/sistema cadastral.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="enter_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255, 255, 255, 0);
border-radius: 100px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="data_nascimento">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>389</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255,255,255, 0); border:  rgba(255,255,255, 0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erro">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>436</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>168</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255,255,255,0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cpf">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255,255,255,0)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
